### ISSUE TITLE
Proximity displaying wrong on Ranked servers

### ISSUE THREAD

**Issue by 2000thehecf1** [OvercastNetwork/Issues#2085]
As you can see here: https://gyazo.com/341595c361cc898e097d5eea3d8ab722 Red and Blue have a very low proximity to the objectives, which is basically impossible when it's only been 2 seconds of the match. I'm pretty sure this has something related with TAB bug too.

**Comment by Electroid** [comment]
Can't reproduce this. Notice anything odd that happened before the match? @2000thehecf1

**Comment by 2000thehecf1** [comment]
@Electroid Nope, as far as i have seen this happens randomly. I've seen this happening again a few more times on the last days and I can't figure why.
